### ISSUE TITLE
Don't getCollectionNames work with Mongo 3?

### ISSUE THREAD

**Issue by pabloalmunia** [mafintosh/mongojs#243]
This code don't return items and I have a lot of collections in my database:

```
db.getCollectionNames(function(err, items) {
        if (err) {
            console.error(err);
            return;
        }
        console.dir(items);
    });

```

**Comment by saintedlama** [comment]
Which node.js and mongojs version are you using? Works in my dev environment.

**Comment by pabloalmunia** [comment]
Node 0.12.5
MongoJS 1.2.1
MongoDB 3.0.4

**Comment by crishushu** [comment]
In my case it even produces an error:

     node_modules/mongojs/node_modules/mongodb/lib/mongodb/connection/base.js:246
             throw message;      
                   ^
     TypeError: Cannot call method 'filter' of undefined
         at /Users/chsc03/DriveDesktop/Projects/MultiRep/GIT/MultiChannel/base/node_modules/mongojs/node_modules/mongodb/lib/mongodb/db.js:469:49
         at /Users/chsc03/DriveDesktop/Projects/MultiRep/GIT/MultiChannel/base/node_modules/mongojs/node_modules/mongodb/lib/mongodb/db.js:1183:7
         at /Users/chsc03/DriveDesktop/Projects/MultiRep/GIT/MultiChannel/base/node_modules/mongojs/node_modules/mongodb/lib/mongodb/db.js:1916:9
         at Server.Base._callHandler (/Users/chsc03/DriveDesktop/Projects/MultiRep/GIT/MultiChannel/base/node_modules/mongojs/node_modules/mongodb/lib/mongodb/connection/base.js:448:41)
         at /Users/chsc03/DriveDesktop/Projects/MultiRep/GIT/MultiChannel/base/node_modules/mongojs/node_modules/mongodb/lib/mongodb/connection/server.js:481:18
         at MongoReply.parseBody (/Users/chsc03/DriveDesktop/Projects/MultiRep/GIT/MultiChannel/base/node_modules/mongojs/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)

**Comment by angelo0000** [comment]
Did you ever find a work around for this?  I am using mongojs 0.18.2 and I am having the same problem.  I get this error when trying to create a new collection against mongo 3.0.7.

**Comment by saintedlama** [comment]
Should be fixed in version 2.0 (released just some seconds ago)

**Comment by saintedlama** [comment]
Any update on this issue?
